ThunderCloud/TSCLink+Localised.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  TSCLink+Localised.swift
+//  ThunderCloud
+//
+//  Created by Joel Trew on 18/09/2017.
+//  Copyright © 2017 threesidedcube. All rights reserved.
+//
+
+import Foundation
+
+public extension TSCLink {
+    
+    /// Helper to initialise a localised link. This is called in TSCLink's initWithDictionary. The method will attempt to correctly initialise the link's url property from the dictionary by matching against the current language.
+    ///
+    /// The method will attempt to find a full match first i.e if the users language is usa_eng it will try to match that, failing that it will fall back to basic eng, and finally fall back to the first url in the links array.
+    ///
+    ///
+    /// - Parameter dictionary: The initialisation dictionary passed from initWithDictionary
+    public func linkFromLocalisedLink(with dictionary: [AnyHashable: Any]) {
```

Comment:
I would expect this to return something given the method name :) I'd rename this to something more suitable maybe, like:

`localise(with dictionary: [AnyHashable: Any])`

Reply:
Whoops I meant to use localise not localised! Fixed, thanks Simon 👍